Assets/scss/components/_home-quote.scss

```diff
@@ -0,0 +1,192 @@
+.component-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    font-family: $openSans;
+
+    @include max-screen(lg) {
+        flex-direction: column;
+    }
+
+    .text-area {
+        width: 33%;
+        padding: toRem(96);
```

Comment:
Could still use padding for left/right, but could use flex instead for vertical position and a min-height on the whole component or the image side. Not necessary to change, just another note for alternate ideas.